be/src/exprs/table_function/java_udtf_function.h

```diff
@@ -0,0 +1,21 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include "exprs/table_function/table_function.h"
+
+namespace starrocks::vectorized {
+
+// Now UDTF only support one column return
+class JavaUDTFFunction final : public TableFunction {
+public:
+    JavaUDTFFunction() = default;
+    ~JavaUDTFFunction() = default;
+
+    Status init(const TFunction& fn, TableFunctionState** state) const override;
```

Comment:
Why need init and prepare method at the same time？

Reply:
init is mainly used for memory allocation and some simple initialisation operations.
open is mainly used to initialize the ClassLoader.
The prepare phase is not very meaningful at the moment and can be removed.